### DISCUSSION TITLE
Does open bidding with AdMob is supported?

### DISCUSSION THREAD

**Discussion by jeremyhalin** [invertase/react-native-firebase#5148] [Q&A] [answered]
I activated it on my AdMob account but ads seem to not load at all. How can I debug this?
On my AdMob account everything is OK and it's working fine with testings IDs. 

Does anyone have open bidding enabled and can confirm it works with this lib?

Thanks!

**Comment by tarouboy** [comment]
I tried some different ways and still couldn't make it work yet.

**Comment by manuhook** [comment, marked as answer]
Yes it's working fine, tested for the moment with Facebook, Index Exchange, Media.net, Rubicon & Pubmatic. You have to sign a contract with each companies. With Facebook you also to have to include a SDK.
